testing-suite/staking-v4/config.py

```diff
@@ -8,6 +8,9 @@
 
 DEFAULT_PROXY = PROXY_CHAIN_SIMULATOR
 
+# TEMP
+OBSERVER_META = "http://localhost:55802"
```

Comment:
I don't think that taking the ports form logs (that sometimes will be saved, or sometimes will be not saved)  is a good solution either. 
This will be solved with another PR inside chain simulator. We will add a endpoint that returns all observers port and I will do another PR with implementation of getting the ports in the right way. For the moment please let this at it is.